packages/text-field/Input.tsx

```diff
@@ -163,16 +166,24 @@ export default class Input<
     }
 
     if (value !== prevProps.value) {
```

Comment:
Let's reopen it, cause I think this is where the change should happen. I think this `if` block should look like this: 
```
if (value !== prevProps.value) {
      const inputValue = this.valueToString(value);
      const notTriggeredChange = !this.state.wasUserTriggeredChange;
      // only call #foundation.setValue on programatic changes;
      // not changes by the user.
      if (notTriggeredChange) {
        foundation && foundation.setValue(inputValue);
      }

      this.setState({wasUserTriggeredChange: false});
    }
```

Reply:
I reflected it!
Is handleValueChange a breaking change?
I saw and removed a test code that checks call count of handleValueChange.

Reply:
Can you remove handleValueChange from everywhere? https://github.com/material-components/material-components-web-react/pull/934/files#diff-0eddc9611a42aba741f3ee96c819757eR31

This is a "breaking change" however no one should have been using this.